documentation/docs/steps/abapEnvironmentRunAUnitTest.md

```diff
@@ -0,0 +1,144 @@
+
+### AUnit test run via Cloud Foundry Service Key example in Jenkinsfile
+
+The following example triggers an AUnit test run via reading the Service Key of an ABAP instance in Cloud Foundry.
+
+You can store the credentials in Jenkins and use the cfCredentialsId parameter to authenticate to Cloud Foundry.
+The username and password to authenticate to ABAP system will then be read from the Cloud Foundry service key that is bound to the ABAP instance.
+
+This can be done accordingly:
+
+```groovy
+abapEnvironmentRunAUnitTest(
+    cfApiEndpoint : 'https://test.server.com',
+    cfOrg : 'cfOrg',
+    cfSpace: 'cfSpace',
+    cfServiceInstance: 'myServiceInstance',
+    cfServiceKeyName: 'myServiceKey',
+    abapCredentialsId: 'cfCredentialsId',
+    aUnitConfig: 'aUnitConfig.yml',
+    script: this,
+)
+```
+
+To trigger the AUnit test run an AUnit config file `aUnitConfig.yml` will be needed. Check the section 'AUnit config file example' for more information.
+
+### ATC run via direct ABAP endpoint configuration in Jenkinsfile
+
+This  example triggers an ATC run directly on the ABAP endpoint.
+
+In order to trigger the ATC run you have to pass the username and password for authentication to the ABAP endpoint via parameters as well as the ABAP endpoint/host. You can store the credentials in Jenkins and use the abapCredentialsId parameter to authenticate to the ABAP endpoint/host.
+
+This must be configured as following:
+
+```groovy
+abapEnvironmentRunATCCheck(
+    abapCredentialsId: 'abapCredentialsId',
+    host: 'https://myABAPendpoint.com',
+    aUnitConfig: 'aUnitConfig.yml',
+    script: this,
+)
+```
+
+To trigger the ATC run an ATC config file `aUnitConfig.yml` will be needed. Check section 'ATC config file example' for more information.
+
+### AUnit config file example
+
+The following section contains an example of an `aUnitConfig.yml` file.
+This file must be stored in the same Git folder where the `Jenkinsfile` is stored to run the pipeline. This repository containing the `Jenkinsfile` must be taken as a SCM in the Jenkins pipeline to run the pipeline.
+
+You can specify an object set containing the objects that should be checked. These can be for example packages, classes or transport requests containing test classes that can be executed. This must be in the same format as below example for a `aUnitConfig.yml` file.
+Note that if you specify a package inside a packageSet to be checked for each package that has to be checked you can configure if you want the subpackages to be included in checks or not.
+
+See below example for an `aUnitConfig.yml` file containing a package to be checked:
+
+```yaml
+title: My AUnit run
+context: My unit tests
+options:
+  measurements: none
+  scope:
+    owntests: true
+    foreigntests: true
+  riskLevel:
+    harmless: true
+    dangerous: true
+    critical: true
+  duration:
+    short: true
+    medium: true
+    long: true
+objectset:
+  - type: unionSet
```

Comment:
Form this docu alone I would not understand what a "unionset" is. We need to link the the official API docu